app/javascript/react/components/FlavorInformationComponent.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const FlavorInformationComponent = (props) => {
+  const { name, description, image_url } = props;
```

Comment:
I actually dig this, but lets go a step further.
- perform the object destructuring  on line 3 where (props) is:
`const FlavorInformationComponent = ({ name, description, image_url }) => {`

Reply:
Made recommended change